content/applications/sales/crm/acquire_leads/send_quotes.rst

```diff
@@ -1,34 +1,149 @@
+   :menuselection:`Sales app --> Configuration --> Sales Orders --> Quotation Templates`. Click
+   on a template to open it, and update the number in the :guilabel:`Quotation Validity` field.
+
+Order lines
+-----------
+
+After updating the customer, payment, and deadline information on the new quotation, the
+:guilabel:`Order Lines` can be updated with the appropriate product information.
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a product`. Type the name of an item into
+the :guilabel:`Product` field to search through the product catalog. Select a product from the
+drop-down menu, or create a new one by selecting :guilabel:`Create` or :guilabel:`Create and Edit`.
+After selecting a product, update the :guilabel:`Quantity` if necessary. Confirm the information in
+the remaining fields.
+
+To remove the line from the quotation, click the :guilabel:`🗑️ (trash can)` icon.
+
+Repeat the steps above until the quotation is complete.
+
+Preview and send
+================
+
+To see a preview of the quotation as the customer will see it, click :guilabel:`Preview`. Doing so
+opens a preview in the :guilabel:`Customer Portal`.
+
+After reviewing the customer preview, click :guilabel:`Return to edit mode`.
+
+When the quotation is ready to deliver to the customer, click :guilabel:`Send by Email`. This opens
+a pop-up window with a pre-configured email message. Information from the quotation, including the
+contact information, total cost, and quotation title will be imported from the quotation. A PDF of
+the quotation is added as an attachment to the email.
+
+.. note::
+   A pre-loaded template is used to create the email message. To alter the template, click the
+   internal link to the right of the :guilabel:`Load template` field. To select a new template,
+   select an option from the :guilabel:`Load template` drop-down.
+
+Make any necessary changes to the email, then click :guilabel:`Send`. A copy of the message is added
+to the *Chatter* of the of the record.
+
+.. seealso::
+   - `Quotation templates </applications/sales/sales/send_quotations/quote_template/>`_
+   - `Optional products </applications/sales/sales/send_quotations/optional_products/>`_
+   - `Quotation deadlines </applications/sales/sales/send_quotations/deadline/>`_
+
+Mark an opportunity won or lost
+===============================
+
+In order to keep the pipeline up to date and accurate, opportunities need to be identified as won
+or lost once a customer has responded to a quotation.
+
+From the quotation form, return to the opportunity using the breadcrumbs at the top left of the
+record. Or navigate to :menuselection:`CRM app --> Sales --> My Pipeline` and click on the correct
+opportunity to open it.
+
+At the top left of the form, click on either :guilabel:`Won` or :guilabel:`Lost`.
+
+If the opportunity it marked *Won*, a green *Won* banner is added to the record, and it is moved to
+the :guilabel:`Won` stage.
```

Comment:
```suggestion
If the opportunity is marked *Won*, a green :guilabel:`Won` banner is added to the record, and it is
moved to the :guilabel:`Won` stage.
```
Used a GUI here because the word "WON" actually appears on the UI.